src/components/ProtectedRoutes.tsx

```diff
@@ -0,0 +1,7 @@
+import { Navigate, Outlet } from "react-router-dom";
+import { useCurrentUser } from "../hooks/hooks";
+
+export const ProtectedRoutes = () => {
```

Comment:
```suggestion
export const ProtectedRoute = () => {
```
Не знаю, насколько уместно тут множественное число.